constants/mapping.go

```diff
@@ -32,3 +32,33 @@ func MetadataKeys() func(string) string {
 		return innerMap[key]
 	}
 }
+
+func DeploymentMetadataKeys() func(string) string {
+	// innerMap is captured in the closure returned below
+	innerMap := map[string]string{
+		"label":           "sw.kogito.kie.org/label",
+		"serviceType":     "sw.kogito.kie.org/name",
+		"description":     "sw.kogito.kie.org/description",
+		"annotations":     "sw.kogito.kie.org/annotations",
+		"dataInputSchema": "sw.kogito.kie.org/dataInputSchema",
+		"expressionLang":  "sw.kogito.kie.org/expressionLang",
+		"metadata":        "sw.kogito.kie.org/metadata",
+		"version":         "sw.kogito.kie.org/version",
+	}
+
+	return func(key string) string {
+		return innerMap[key]
+	}
+}
+
+func PlatformAnnotation() func(string) string {
+	// innerMap is captured in the closure returned below
+	innerMap := map[string]string{
+		"SecondaryPlatformAnnotation": "sw.kogito.kie.org/secondary.platform",
```

Comment:
what is this "SecondaryPlatformAnnotation" proeprty?

Reply:
It is an annotation used to support a secondary platform in Ready state that it is not the default one.

In these way you could have 2 platforms in Ready state (for example Kubernetes and Knative) and deploy by default on one of them but if you would like to deploy on the other one you can explicitly choose it.

At the moment there is something missing to support this use case, but the API are ready.

Reply:
OK, cool, when possible, please add some info in the methods :)